.github/workflows/tests.yml

```
@@ -71,15 +75,15 @@ jobs:
       # DEBT-123 AC9: Codecov upload — intentionally non-blocking (external service, not security)
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v4
-        if: matrix.python-version == '3.11'
+        if: true
```

actionlint (1.7.12)

[error] constant expression "true" in condition. remove the if: section

(if-cond)